src/main/groovy/org/jetbrains/intellij/PatchPluginXmlTask.groovy

```diff
@@ -147,4 +159,15 @@ class PatchPluginXmlTask extends ConventionTask {
             }
         }
     }
+
+    static void patchChangeNotes(String changeNotes, Node pluginXml) {
```

Comment:
It's very similar to `patchDescription`, right? Maybe we can generalize them somehow?